app/views/shared/_citations.html.erb

```diff
@@ -15,18 +15,18 @@
 <% end %>
 
 <% content_for(:gscholar_meta) do %>
-  <meta name="citation_title" content="<%= @presenter.title.first %>"/>
-  <% @presenter.creator.each do |creator| %>
-  <meta name="citation_author" content="<%= creator %>"/>
-  <% end %>
-  <meta name="citation_publication_date" content="<%= @presenter.date_created.first %>"/>
-  <meta name="citation_pdf_url" content="<%= main_app.download_url(@presenter) %>"/>
-  <!-- Sufia does not yet support these metadata -->
-  <!--
-    <meta name="citation_journal_title" content=""/>
-    <meta name="citation_volume" content=""/>
-    <meta name="citation_issue" content=""/>
-    <meta name="citation_firstpage" content=""/>
-    <meta name="citation_lastpage" content=""/>
-  -->
+    <meta name="citation_title" content="<%= @presenter.title.first %>"/>
```

Comment:
Please use 2-space indentation style per http://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#follow-the-coding-conventions